packages/predicates/README.md

```diff
@@ -0,0 +1,31 @@
+# `@shopify/predicates`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fpredicates.svg)](https://badge.fury.io/js/%40shopify%2Fpredicates.svg)
+
+A set of common predicates for JS/TS apps. Most of the functions in this library either take one input and return `true` / `false`, or return a customized function that does so.
```

Comment:
Remove `most of`

Reply:
Also if go with the description change I suggested, replace this first sentence.